soc/openhwgroup/cva6/CMakeLists.txt

```diff
@@ -3,6 +3,6 @@
 
 add_subdirectory(${SOC_SERIES})
 
-zephyr_library_sources_ifdef(CONFIG_SOC_FAMILY_CVA6_PROVIDE_NONSTANDARD_CACHE_OPTIONS soc_cache_management.c)
+zephyr_library_sources_ifdef(CONFIG_CACHE_MANAGEMENT soc_cache_management.c)
```

Comment:
Question: `CONFIG_CACHE_MANAGEMENT` doesn't seem to be selected by default - is this intentional?

Reply:
The option is off by default in order to reduce code size if the cache API is not used.
I have seen that subsystems that call cache management functions such as LLEXT enable `CONFIG_CACHE_MANAGEMENT` as a dependency.

Reply:
Got it